app/helpers/dashboard_helper.rb

```diff
@@ -0,0 +1,2 @@
+module DashboardHelper
+end
```

Comment:
```suggestion
```

Reply:
Reason is because Helper files actually aren't _necessary_ unless you have some function you want to add to them.